alicloud/table_alicloud_rds_backup.go

```diff
@@ -0,0 +1,275 @@
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "backup_id", Require: plugin.Optional},
+				{Name: "db_instance_id", Require: plugin.Optional},
+				{Name: "backup_status", Require: plugin.Optional},
+				{Name: "backup_mode", Require: plugin.Optional},
+				{Name: "backup_start_time", Require: plugin.Optional, Operators: []string{"="}},
+				{Name: "backup_end_time", Require: plugin.Optional, Operators: []string{"="}},
+				{Name: "backup_location", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: []*plugin.Column{
+			// Top columns
+			{
+				Name:        "backup_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the backup set.",
+			},
+			{
+				Name:        "db_instance_id",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DBInstanceId"),
+				Description: "The ID of the single instance to query.",
+			},
+			{
+				Name:        "backup_status",
+				Description: "The status of the backup. Valid values: Success, Failed.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "backup_mode",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup mode.",
+			},
+			{
+				Name:        "backup_db_names",
+				Type:        proto.ColumnType_STRING,
+				Description: "The name of the database that has been backed up.",
+			},
+			{
+				Name:        "backup_size",
+				Type:        proto.ColumnType_INT,
+				Description: "The size of the backup set. Unit: bytes.",
+			},
+			{
+				Name:        "backup_start_time",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The beginning of the backup time range. The time is in the yyyy-MM-ddTHH:mm:ssZ format and displayed in UTC.",
+			},
+			{
+				Name:        "backup_end_time",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The end of the backup time range. The time is in the yyyy-MM-ddTHH:mm:ssZ format and displayed in UTC.",
+			},
+			{
+				Name:        "backup_download_url",
+				Type:        proto.ColumnType_STRING,
+				Description: "The Internet download URL of the backup set. If the download URL is unavailable, this parameter is an empty string.",
+				Transform:   transform.FromField("BackupDownloadURL"),
+			},
+			{
+				Name:        "backup_extraction_status",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup extraction status.",
+			},
+			{
+				Name:        "backup_intranet_download_url",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("BackupIntranetDownloadURL"),
+				Description: "The internal download URL of the backup set.",
+			},
+			{
+				Name:        "backup_method",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup method. Valid values: Snapshot, Physical and Logical.",
+			},
+			{
+				Name:        "backup_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup method.",
```

Comment:
```suggestion
				Description: "The backup type.",
```

Please include a set of valid values, if available.